database/queries.ts

```diff
@@ -31,7 +31,10 @@ export const setupDatabase = async () => {
       InventoryOfItems TEXT
     );
     INSERT INTO App (StreakLastChecked, Streak, CurrencyAmount, InventoryOfItems)
-    VALUES (-1, 0, 100, '[]');
+    VALUES (-1,
+      0,
+      100,
+      '[{"item": {"id": 1, "name": "apple", "cost": 10, "icon": "🍎"}, "count": 1}]');
     DROP TABLE IF EXISTS Item_json;
     CREATE TABLE IF NOT EXISTS Item_json (
       items TEXT
```

Comment:
This schema seems a little weird. We can have multiple rows of inventory items (each row defining a list of items). Multiple rows of streaks and currency amounts also seems odd.